### DISCUSSION TITLE
npm install latest misses some folders

### DISCUSSION THREAD

**Discussion by djcaesar9114** [socketio/socket.io#3813] [Q&A] [answered]
Hello,

I tried to install "socket.io-client". When I write `npm i -S socket.io-client`, it installs the version 2.4.0 and everything works fine, except that I need the latest version of this package to be able to communicate with the server.

I tried `npm i -S socket.io-client@latest`, it installs the latest version, 3.1.1, but with some folders missing, such as lib, and I need all the folders to have the package working, logically.

How can I install the latest version of the package? Other question: why doesn't it install the latest version if I don't precise it?

**Comment by darrachequesne** [comment, marked as answer]
That's weird, I'm not able to reproduce the behavior you are seeing:

```
$ npm info socket.io-client

socket.io-client@3.1.1 | MIT | deps: 7 | versions: 95
[...]

dist-tags:
beta: 3.0.0-rc4   latest: 3.1.1     v2-latest: 2.4.0
```

Could you please check?
